apps/backend/api/models/Notification.js

```diff
@@ -434,7 +427,7 @@ module.exports = bookshelf.Model.extend({
         post_title: decode(post.title() || ''),
         post_topic: firstTag,
         post_type: post.get('type'),
-        post_url: Frontend.Route.tokenLogin(reader, token, this.postUrlHelper({ post, isPublic: false, topic: firstTag, group }) + '?ctt=post_email&cti=' + reader.id),
+        post_url: Frontend.Route.tokenLogin(reader, token, Frontend.Route.post(post, group, 'ctt=post_email&cti=' + reader.id)),
```

Comment:
Do you think this fixes https://github.com/Hylozoic/hylo/issues/183 ?

Reply:
hmm, i doubt it, that sounds like a front end issue. but its not happening for me